labrad/types.py

```diff
@@ -295,8 +295,20 @@ def unflatten(s, t, endianness='>'):
     return t.__unflatten__(s, endianness)
 
 
-FlatData = collections.namedtuple('FlatData', ['bytes', 'tag'])
+class FlatData(collections.namedtuple('FlatDataBase', ['bytes', 'tag', 'endianness'])):
+    """FlatData represents unflattened data.
 
+    The elements are
+
+    bytes:      The data as intended to be transmitted over the wire
+    tag:        The LabRAD data type in the form of an LRType object.
```

Comment:
As I mentioned above, it would be good here to align with the distinction between tag (string) and type (LRType), but not sure the right way to do that, while keeping the `namedtuple` semantics. Maybe keep the parsed type as a `.t` field, and make `tag` be a proprty that returns a string?